lib/evmone/eof.hpp

```diff
@@ -15,6 +15,7 @@
 
 namespace evmone
 {
+inline uint16_t read_uint16_be(auto it) noexcept;
```

Comment:
Move both of these helper functions here (`read_int16_be` and `read_uint16_be`).